dist/src/decoder.js

```diff
@@ -1,5 +1,6 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
+exports.decode = void 0;
```

Comment:
what is dist?

Reply:
It's in the dist folder and generated but it's an interesting row

Reply:
And overwritten on line 6 :P

Reply:
haha yes I see it